lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
I see we have a number of common templates, like ``mathjax_include.html``. Maybe now would be the right time to clean up the segment code to be in common too.